src/components/experimental/TextField/ClearButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import { Button as BaseButton, ButtonProps as BaseButtonProps } from 'react-aria-components';
+import { getSemanticValue } from '../../../essentials/experimental/cssVariables';
+import { XCrossCircleIcon } from '../../../icons';
+
+const StyledButton = styled(BaseButton)`
+    appearance: none;
+    background: none;
+    border: none;
+    display: flex;
+    margin: 0;
+    padding: 0;
+    color: ${getSemanticValue('on-surface-variant')};
```

Comment:
What does the `-variant` suffix mean?

Reply:
Hmm, like an alternate version.... 

I'd talk about this with Marian... it seems that meaning is a bit subjective. Probably there's an explanation... but seems that the explanation would be needed, if someone would autonomously combine this colours (without a design spec sheet).